source/basic.tex

```diff
@@ -6351,7 +6350,7 @@
 guarantees, which in turn are stronger than weakly parallel forward progress
 guarantees.
 \begin{note}
-For example, some kinds of synchronization between threads of execution might only
+For example, some kinds of synchronization between threads of execution would only
 make progress if the respective threads of execution provide parallel forward progress
```

Comment:
I feel like this should now say "if the respective threads ... _provided_ parallel forward progress guarantees." What do you think?

Reply:
of course
fixed